config/locales/views/submissions/en.yml

```diff
@@ -55,6 +55,7 @@ en:
                                 Make sure you have worked on the most recent files before submitting."
 
     # Miscellaneous
+    files: "Solution Files"
```

Comment:
put this in `locales/views/automated_tests` and name it something like `solution_files`, just `files` is not descriptive enough